PicPi/AddPhotoViewController.swift

```diff
@@ -0,0 +1,196 @@
+//
+//  AddPhotoViewController.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 12/15/20.
+//
+import Photos
+import Moya
+import UIKit
+import BSImagePicker
+
+class AddPhotoViewController: UIViewController {
+  
+  let sendButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  let photoUploadManager = PhotoUploadManager()
+  let doneButton = customBtnRoundCornerBlueWithShadow(type: .custom)
+  
+  private let tableView = UITableView()
+  private var margins: UILayoutGuide!
+  
+  override func loadView() {
+    super.loadView()
+    view.backgroundColor = .white
+    margins = view.layoutMarginsGuide
+    setupSendButton()
+    setupDoneBtn()
+    setupTableView()
+    photoUploadManager.delegate = self
+    
+  }
+  
+  // MARK: - UI Setup
+  private func setupSendButton() {
+    
+    view.addSubview(sendButton)
+    sendButton.addTarget(self, action: #selector(sendButtonPressed), for: .touchUpInside)
+    sendButton.setTitle("Send", for: .normal)
+    sendButton.setTitleColor(.darkGray, for: .disabled)
+    sendButton.isEnabled = false
+    sendButton.sizeToFit()
+    sendButton.translatesAutoresizingMaskIntoConstraints = false
+    NSLayoutConstraint.activate(
+      [
+        sendButton.centerXAnchor.constraint(equalTo: margins.centerXAnchor) ,
+        sendButton.centerYAnchor.constraint(equalTo : margins.bottomAnchor, constant:  -sendButton.frame.height)  ,
```

Comment:
nit: line break this like so
```suggestion
        sendButton.centerYAnchor.constraint(
          equalTo: margins.bottomAnchor, 
          constant: -sendButton.frame.height
        ),
```